docs/user-guides/workspace-access/zed.md

```diff
@@ -62,6 +62,19 @@ Use the Coder CLI to log in and configure SSH, then connect to your workspace wi
    zed ssh://coder.workspace-name
    ```
 
+1. Connect via a coder_app workspace icon and SSH hotlink:
+
+   ```hcl
+   resource "coder_app" "zed" {
+     agent_id = coder_agent.main.id
+     slug          = "slug"
+     display_name  = "Zed"
+     external = true
+     url      = "zed://ssh/coder.${data.coder_workspace.me.name}"
+     icon     = "/icon/zed.svg"
+   }
+   ```
```

Comment:
As this is terraformed code, it should go somewhere into the https://coder.com/docs/admin/templates/extending-templates section. User guides are for developers with member roles who may not have template admin roles to actually add something to a template.

Reply:
good call, @matifali - I'll take care of that